bentoctl_aws_ec2/deploy.py

```diff
@@ -48,13 +47,15 @@ def deploy(bento_bundle_path, deployment_name, ec2_config):
 
     with console.status("Building image"):
         repository_id, registry_url = create_ecr_repository_if_not_exists(
-            ec2_config["region"], repo_name
+            deployment_spec["region"], repo_name
+        )
+        _, username, password = get_ecr_login_info(
+            deployment_spec["region"], repository_id
         )
-        _, username, password = get_ecr_login_info(ec2_config["region"], repository_id)
         ecr_tag = generate_docker_image_tag(
-            registry_url, bento_metadata.name, bento_metadata.version
+            registry_url, bento_tag.name, bento_tag.version
```

Comment:
let's get this info from asking user, rather than construct it from the bento info

Reply:
We can have the default to bento tag

Reply:
Let's add the docker_image_tag as part of the deployment spec as an optional field, with the default to bento tag

Reply:
the default will be {bento-name}:{bento-version}